pkg/rekor/identity.go

```diff
@@ -72,16 +78,21 @@ type MonitoredValues struct {
 	// Subjects contains a list of subjects that are not specified in a
 	// certificate, such as a SSH key or PGP key email address
 	Subjects []string `yaml:"subjects"`
+	// OIDMatchers contains a list of OID extension fields and associated values
+	// ex. Build Signer URI, associated with specific workflow URIs
+	OIDMatchers []OIDMatcher `yaml:"oidMatchers"`
 }
 
 // IdentityEntry holds a certificate subject, issuer, and log entry metadata
 type IdentityEntry struct {
-	CertSubject string
-	Issuer      string
-	Fingerprint string
-	Subject     string
-	Index       int64
-	UUID        string
+	CertSubject    string
+	Issuer         string
+	Fingerprint    string
+	Subject        string
+	Index          int64
+	UUID           string
+	OIDExtension   asn1.ObjectIdentifier
+	ExtensionValue string
 }
 
 func (e *IdentityEntry) String() string {
```

Comment:
Note: we need to extend this to include the new fields (see the range in line 89). I think it's ok to do this in a new PR

Reply:
creating a separate PR for this!